cypress/support/custom_commands/login.helper.js

```diff
@@ -125,43 +125,8 @@ const fillLoginForm = (username, password) => {
 	);
 };
 
-const shuffleString = (str) => {
-	const characters = str.split("");
-	let currentIndex = characters.length;
-	let randomIndex;
-
-	while (currentIndex !== 0) {
-		randomIndex = Math.floor(Math.random() * currentIndex);
-		currentIndex--;
-		[characters[currentIndex], characters[randomIndex]] = [
-			characters[randomIndex],
-			characters[currentIndex],
-		];
-	}
-
-	return characters.join("");
-};
-const generateStrongPassword = (length) => {
-	const lowercase = "abcdefghijklmnopqrstuvwxyz";
-	const uppercase = lowercase.toUpperCase();
-	const numbers = "0123456789";
-	const symbols = "!@#$%^&*";
-
-	const allChars = lowercase + uppercase + numbers + symbols;
-
-	let password = "";
-	for (let i = 0; i < length; i++) {
-		const charSet = allChars[Math.floor(Math.random() * allChars.length)];
-		const randomIndex = Math.floor(Math.random() * charSet.length);
-		password += charSet[randomIndex];
-	}
-
-	return shuffleString(password);
-};
-
 const studentFirstLogin = (environment) => {
-	const newPassword = generateStrongPassword(12);
-	Cypress.env("password", newPassword);
+	Cypress.env("password", "SET_NEW_PWD_BY_STUDENT");
```

Comment:
```suggestion
	Cypress.env("password", Cypress.env("SET_NEW_PWD_BY_STUDENT"));
```
Otherwise you set it to the string "SET_NEW_PWD_BY_STUDENT", don't you?

Reply:
So it will not work like that. See this [Override-multiple-values-from-the-Cypress-configuration-by-passing-an-object](https://docs.cypress.io/api/cypress-api/env#Override-multiple-values-from-the-Cypress-configuration-by-passing-an-object)

since in our env file we already have `SET_NEW_PWD_BY_STUDENT` variable so we just have to set it to a `password` variable so we can use it. Alternatively, we can do something like that but its more or less the same thing

```js
const password = Cypress.env("SET_NEW_PWD_BY_STUDENT")
cy.get(`randomSelector`).type(password)
```

Reply:
I think you are confusing something here. `Cypress.env("password", Cypress.env("SET_NEW_PWD_BY_STUDENT"));` does not **set** "SET_NEW_PWD_BY_STUDENT"; it **gets** its value and then **sets** "password".
It is the same as:
```
const password = Cypress.env("SET_NEW_PWD_BY_STUDENT")
Cypress.env("password", password);
```

Reply:
Maybe just write it in two lines because it's easier to read.

Reply:
understood adapted it.